src/Websites/Websites/Utilities/WebsitesClient.cs

```diff
@@ -820,6 +819,9 @@ public Site UpdateHostNameSslState(string resourceGroupName, string webAppName,
                 Location = location
             };
 
+            var webapp = GetWebApp(resourceGroupName, webAppName, slotName);
+            webappWithNewSslBinding.Tags = webapp.Tags;
```

Comment:
@panchagnula we should include a null check for the returned `webapp`, or would this fail somewhere before if the web app doesn't exist?

Reply:
since the command is run on a webapp context webpp by itself will not be null